### DISCUSSION TITLE
Export and import database as an excel between different projects

### DISCUSSION THREAD

**Discussion by muditchordia** [LCA-ActivityBrowser/activity-browser#1325] [Q&A]
Hi,

I am trying to export a user created database from an older project (Pro_1) to use it for a new project (Pro_2) as an excel file. Pro_1 was based on e3.91 and Pro_2 on e3.10. When I export the excel from Pro_1 I change the reference to technosphere to the new ecoinvent database name in Pro_2. When I try to import this excel file, the import seems to go into a loop [writing datasets to SQLite database] and it does not move forward.

Question: 

- Is it even possible to export from a project and import to another project? Does it have to do with dependencies such as the codes with ecoinvent versions?
- If it is indeed possible, what am I doing incorrectly?

Thanks 

Mudit

**Comment by marc-vdm** [comment]
Closing, discussed in RL
